.github/workflows/publish-docker.yaml

```diff
@@ -0,0 +1,52 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: publish-docker
+
+on:
+  push:
+    branches:
+      - master
+
+env:
+  HUB: ghcr.io/apache/skywalking-python
+  PROJECT: skywalking-python
+
+jobs:
+  build-docker:
+    if: github.repository == 'apache/skywalking-python'
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    timeout-minutes: 30
+    env:
+      tag: ${{ github.sha }}
```

Comment:
... and this is not used

Reply:
Ah, my mistake. I thought we had to adhere to the tag naming convention used by Docker Hub's.
So, if I get this right, you mean to prefix the tag with the commit SHA, like skywalking-python:e1a02359b239bd28de3f6d35fdc870250fa513d5-1.0.0-kafka-py3.10?

Reply:
> Ah, my mistake. I thought we had to adhere to the tag naming convention used by Docker Hub's.
> So, if I get this right, you mean to prefix the tag with the commit SHA, like `skywalking-python:e1a02359b239bd28de3f6d35fdc870250fa513d5-1.0.0-kafka-py3.10`?

Yeah but not quite, we don't need `1.0.0` here, just use `skywalking-python:e1a02359b239bd28de3f6d35fdc870250fa513d5-kafka-py3.10`, as the sha acts as a "version" number